modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -408,9 +408,20 @@ private KeyLocalizedLabelPair _getKeyLocalizedLabelPair(
 	}
 
 	private com.liferay.object.rest.dto.v1_0.ObjectEntry _getObjectEntry(
-		String externalReferenceCode, ObjectDefinition objectDefinition,
+		ObjectDefinition objectDefinition, ObjectEntry objectEntry,
 		ThemeDisplay themeDisplay) {
 
+		if (objectEntry instanceof ProxyObjectEntry) {
+			ProxyObjectEntry proxyObjectEntry = (ProxyObjectEntry)objectEntry;
+
+			com.liferay.object.rest.dto.v1_0.ObjectEntry restObjectEntry =
+				proxyObjectEntry.getRestObjectEntry();
+
+			if (restObjectEntry != null) {
+				return restObjectEntry;
```

Comment:
@ealonso in this other [code](https://github.com/liferay/liferay-portal/blob/ff7e363eb16ca28825a6071c4e3e9d309c5e1c0c/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFolderResourceImpl.java#L399-L403) we are using the prefix `dto` instead. Can we follow this same pattern here? 

I didn't apply this change yet because I don't know if the rest of the changes can be forwarded (I only reviewed the changes related to the Object's module).

Reply:
Hey @carolmariaabb,

Sure we can use `dto` instead, I've already update it. Also, the other commits were already reviewed, so if everything is ok, you can forward it.